pkg/controller/plan/migration.go

```diff
@@ -1175,6 +1179,12 @@ func (r *Migration) updateCopyProgress(vm *plan.VMStatus, step *plan.Step) (err
 		return
 	}
 
+	if !r.Plan.Spec.Warm {
```

Comment:
Why did you add this here? `updateCopyProgress` is not specific to warm migration, it tracks copy progress for all disk transfers.

Reply:
you're right, this should not be here, I had some applied changes I had to work with and accidentally left this, I'll remove it promptly